.gitignore

```diff
@@ -1,6 +1,7 @@
 .env
 *.pyc
 app/config/local_config.yml
+app/static/css
```

Comment:
This is where our generated SASS goes... It's confusing right now because we also have hand-written CSS in `app/static/styles`, but once we've fully moved over to claudio's CSS, I assume we'll get rid of that.